src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -424,6 +424,17 @@ public function getCMSFields()
                 'WorkerCount',
             ]);
 
+            // Display override to remove the left margin to address a systemic issue with CompositeField
+            // TODO remove this patch after https://github.com/silverstripe/silverstripe-framework/issues/11857 is fixed
+            $cssOverrideContent = <<<'HTML'
+<style>
+.composite,
+.composite > .form__field-holder {
+  margin-left:0 !important;
+}
+</style>
+HTML;
```

Comment:
```suggestion
            <style>
            .composite,
            .composite > .form__field-holder {
              margin-left:0 !important;
            }
            </style>
            HTML;
```
You can keep this indented in modern PHP.